From 2fa057ca311b813b66ba1eacb611683930394d52 Mon Sep 17 00:00:00 2001
From: "Vadim V. Vlasov" <vadim.vlasov@t-platforms.ru>
Date: Wed, 3 Jul 2019 12:40:29 +0300
Subject: [PATCH 9/9] Poweroff instead of reboot on exit

Poweroff seems more common scenario to exit after
all modifications are done.

Signed-off-by: Vadim V. Vlasov <vadim.vlasov@t-platforms.ru>
---
 src/main.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/main.c b/src/main.c
index 27dd7d1..2613aa1 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1,5 +1,4 @@
 #include <unistd.h>
-#include <sys/reboot.h>
 #include <sys/select.h>
 #include <sys/stat.h>
 #include <sys/types.h>
@@ -228,7 +227,7 @@ int main(void) {
     sync();
   }
   sync();
-  reboot(RB_AUTOBOOT);
+  execl("/sbin/poweroff", "poweroff", NULL);
 
 	return 0;
 }
-- 
2.7.4

